qucs-doc/tutorial/textmode/Makefile.am

```diff
@@ -31,7 +31,14 @@ TARGETS = textmode.pdf
 
 export TEXINPUTS=$(srcdir):$(srcdir)/..:
 
-tutorial: $(TARGETS)
+textmode.pdf: $(SOURCES)
+
+pics-tutorial:
```

Comment:
forgot to comment on the `-recursive`, did not think about that, but could likely be useful. But I'm not too familiar with the auto-stuff, so this will be maybe for another PR, after fixing all the other things around...